portfolio/pom.xml

```diff
@@ -38,6 +38,13 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+    
+    <!-- Gson -->
+    <dependency>
```

Comment:
In general you don't want to add dependencies you aren't currently using (the use of gson is commented out), but it's okay since you'll probably use it later anyway